diviner/analysis/pmdarima_analyzer.py

```diff
@@ -261,24 +261,33 @@ def calculate_acf(
                          denominator mode of calculation for the autocovariance of the series.
         :return: Dictionary of ``{<group_key>: {<acf terms>: <values as array>}}``
         """
+
+        import pmdarima
+
         self._create_group_df()
         group_acf_data = {}
         for group, df in self._group_df:
-            acf_data = acf(
-                x=df[self._y_col],
-                unbiased=unbiased,
-                nlags=nlags,
-                qstat=qstat,
-                fft=fft,
-                alpha=alpha,
-                missing=missing,
-                adjusted=adjusted,
-            )
-            group_data = (
-                {"acf": acf_data[0]}
-                if isinstance(acf_data, tuple)
-                else {"acf": acf_data}
-            )
+            if Version(pmdarima.__version__) < Version("2.0.0"):
+                acf_data = acf(  # pylint: disable=E1123
```

Comment:
```suggestion
                acf_data = acf(  # pylint: disable=unexpected-keyword-arg
```

Nit: can we use the error name instead of the error code?

Reply:
great point